components/incidents/incidents-list-view/loadPageData.tsx

```diff
@@ -0,0 +1,134 @@
+import { useEffect, useState } from "react";
+import { NEXT_PUBLIC_AUTH_TOKEN } from "../../../constants";
+import { PageType } from "./incident_list_view";
+
+/**
+ * Loads data from API
+ * @param pageNumber pageNumber required for paginated data
+ * @param pageType type of page
+ * @returns API response and API loading status
+ */
+
+const limit = 10; // determines the no. of rows of data in a page
+
+export default function LoadPageData(pageNumber: number, pageType: string) {
```

Comment:
hook's name should start with use prefix

Reply:
Done